packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -51,7 +51,7 @@ enum AliasState {
     None = "None",
 }
 
-class DataStore implements IDataStore {
+class DataStore implements IDataStore, Partial<IFluidLoadable> {
```

Comment:
this should be Partial<IProvideFluidLoadable>

Reply:
the class is also missing a property:
```
get IFluidLoadable(){
   if(this.handle !== undefined){
      return this;
   }
}
```

Reply:
we don't generally make the implementation partial, only the IProvide. This allows optional delegration to the real thing, but forces the whole implementation interface to be implemented